### ISSUE TITLE
renaming morph:(P/p)aradigm

### ISSUE THREAD

**Issue by matteo-pellegrini** [ontolex/morph#21]
## The issue

In theoretical morphology, the term "paradigm" is used to refer to the set of word forms of a given lexeme -- or, from a more abstract perspective, the set of feature-value specifications for which lexemes of a given lexical category are inflected.
However, currently the class `morph:Paradigm` appears to behave like a set of inflection *rules*, rather than word forms.
Therefore, it seems to correspond to an "inflection class" in morphological terms.
This is also consistent with the current definition

> Class `morph:Paradigm` represents a theoretically motivated type of declension

(although this would probably better be stated more generally rather than only regarding "declension", i.e. nominal inflection)

Below are two alternative proposals that I make to better reflect this state of affairs.

## 1. My proposal

1. 

> `morph:Paradigm` > `morph:InflectionClass`
> `morph:paradigm` > `morph:inflectionClass`

- PRO
    - only a naming change with no consequences on the structure of the model

- CON
    - risk of confusion with `rdf:Class`? (cf. observation by Gilles Sérasset in previous telcos)
    - risk of confusion with `morph:InflectionType`?

## 2. Alternative proposal

> `morph:Paradigm` > `morph:morphologicalPattern`

- PRO
    - consistent with the fact that the class fills the gap in the range of the core ontolex property `ontolex:morphologicalPattern`, whose range was left unspecified, to be specified in more specific vocabularies -- what is now accomplished with Morph

- CON
    - requires thiking about another name for the associated property, that cannot simply become `morphologicalPattern`, as this is already the name of the core ontolex property linking to `ontolex:LexicalEntry`

## A related issue

As hinted above, with the first option there is also some risk of confusion with the already existing Morph class and property `morph:InflectionType` and `morph:inflectionType`, respectively.

Indeed, I personally find also these names to be not immediately suggestive of the behavior of the class and associated property.
Furthermore, there is also the risk of confusion with the already existing class [lila:InflectionType](https://lila-erc.eu/lodview/ontologies/lila/InflectionType) and property [lila:hasInflectionType](https://lila-erc.eu/ontologies/lila/hasInflectionType), that are actually used for types of conjugation/declension in Latin.
Therefore, I would also propose to rename:

> `morph:InflectionClass` > `morph:InflectionSlot`
> `morph:inflectionType` > `morph:inflectionSlot`

which also seems more consistent with the current definition

> Class `morph:InflectionType` represents a single slot for all values of a grammatical category or, in the case of bundles of grammatical categories, for all combinations of values thereof

where the term *slot* is explicitly used.

To summarize, my personal preference is for option 1, i.e. renaming `P/paradigm` to `I/inflectionClass`, with the additional renaming of `I/inflectionType to` `I/inflectionSlot` -- but I am looking forward to hear what your opinions are.

**Comment by chiarcos** [comment]
Proposal accepted, see https://github.com/ontolex/morph/blob/master/doc/wrapup/minutes-2025-06-04.md
